Unity/Assets/CrosshairCursor.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CrosshairCursor : MonoBehaviour
```

Comment:
This should be moved to the `Unity/Assets/Scripts` folder and be in the `LD51.Unity` namespace.